pkg/utils/util.go

```diff
@@ -150,6 +153,12 @@ var SnapshotterSecretParams = secretParamsMap{
 	secretNamespaceKey: PrefixedSnapshotterSecretNamespaceKey,
 }
 
+var GroupSnapshotterSecretParams = secretParamsMap{
+	name:               "Snapshotter",
```

Comment:
Name should `GroupSnapshotter`  as snapshotter already sed for `SnapshotterSecretParams`